src/containerapp/azext_containerapp/custom.py

```diff
@@ -508,7 +508,7 @@ def create_containerapp(cmd,
         if "properties" in r and "provisioningState" in r["properties"] and r["properties"]["provisioningState"].lower() == "waiting" and not no_wait:
             not disable_warnings and logger.warning('Containerapp creation in progress. Please monitor the creation using `az containerapp show -n {} -g {}`'.format(name, resource_group_name))
 
-        if "configuration" in r["properties"] and "ingress" in r["properties"]["configuration"] and "fqdn" in r["properties"]["configuration"]["ingress"]:
```

Comment:
update test for location scenario, too

Reply:
Done.
test_containerapp_up_on_arc_image_e2e check environment location equals connected cluster location.